docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ruby:2.6.5
+
+# Install additional packages
+RUN apt-get update             && \
+    apt-get install -y            \
+        calibre                   \
+        default-mysql-client      \
+        phantomjs
```

Comment:
I missed out zip wkhtmltopdf which are needed for downloads sorry

Reply:
No worries, added and squashed in